apps/src/craft/craft.js

```diff
@@ -161,6 +176,42 @@ Craft.init = function (config) {
   Craft.level = config.level;
   Craft.skin = config.skin;
 
+  var levelTracks = [];
+  if (Craft.level.songs && musicMetadata) {
+    levelTracks = musicMetadata.filter(function(trackMetadata) {
+      return Craft.level.songs.indexOf(trackMetadata.name) !== -1;
+    });
+  }
+
+  Craft.musicController = new MusicController(
+      studioApp.cdoSounds,
+      (filename) => config.skin.assetUrl(`music/${filename}`),
```

Comment:
:innocent: I'll replace with normal callback until we talk about ES6 usage / style next week